kubernetes/platform-services/Chart.yaml

```diff
@@ -25,41 +25,37 @@ appVersion: 0.1.0
 dependencies:
   - name: hypertrace-oc-collector
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.1.5
+    version: 0.1.6
   - name: span-normalizer
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.1.23
+    version: 0.3.0
```

Comment:
All ingestion with the same version. good one.

Reply:
yeah! after `hypertrace-service` it will become more easier!